lib/screens/play/current_card/card_actions/card_action_text.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+class CardActionText extends StatelessWidget {
+  final String actionText;
+
+  CardActionText([this.actionText = '']);
```

Comment:
If we switch to only creating these for concrete actions, I would imagine we could drop this default (as we'll always be passing a populated `actionText`)